administration/serializers.py

```diff
@@ -1,31 +1,11 @@
-from djoser.serializers import UserSerializer
-from django.contrib.auth import get_user_model
-from django.core.exceptions import ObjectDoesNotExist
-from rest_framework.serializers import SerializerMethodField
+from rest_framework import serializers
 
-User = get_user_model()
+from authentication.models import CustomUser
+from profiles.models import Profile
 
 
-class AdminUserSerializer(UserSerializer):
-
-    phone_number = SerializerMethodField()
-
-    def get_phone_number(self, user):
-        try:
-            return user.profile.comp_phone_number
-        except ObjectDoesNotExist:
-            return None
-
-    class Meta(UserSerializer.Meta):
-        model = User
-        fields = (
-            "id",
-            "person_name",
-            "person_surname",
-            "person_email",
-            "phone_number",
-            "last_login",
-            "is_active",
-            "is_staff"
-        )
-        read_only_fields = ("phone_number", "person_email")
+class AdminCompanySerializer(serializers.ModelSerializer):
+    person = serializers.PrimaryKeyRelatedField(queryset=CustomUser.objects.all())
+    class Meta:
+        model = Profile
+        fields = "__all__"
```

Comment:
Again, I don't recommend using `__all__`, explicit fields enumeration is safer.

Reply:
Fixed